### PR TITLE
Build: Remove the rpmlintrc file.

### PR DESCRIPTION
**Pull request by clumens**
First, we were using the wrong command line option.  It should be "-r rpmlintrc".  Second, everything we have in the rpmlintrc file at the moment is not even necessary, causing rpmlint to throw errors about unused filters.

We can later bring the file back if it's necessary, but just removing it causes rpmlint to pass.

Fixes T760

### CODE REVIEW

**Comment by clumens** [comment]
Some of the checks it's performing look like they could still be useful, so alternately we could add `--ignore-unused-rpmlintrc`.

File: rpm/Makefile.am [hunk 1]
```diff
@@ -193,7 +193,7 @@ rpm-clean: spec-clean srpm-clean
 
 .PHONY: rpmlint
 rpmlint: $(RPM_SPEC_DIR)/$(PACKAGE).spec
-	rpmlint -f rpmlintrc "$<"
+	rpmlint "$<"
```
[the remark this answers is not in the document]
Comment (clumens) [reply rpm/Makefile.am:195]: I'm not sure how we can get rpmlint to pass when the invalid-url filter is always enabled.

[the remark this answers is not in the document]
Comment (kgaillot) [reply rpm/Makefile.am:195]: --ignore-unused-rpmlintrc should do it

[the remark this answers is not in the document]
Comment (clumens) [reply rpm/Makefile.am:195]: But not on older versions that don't support it.

[the remark this answers is not in the document]
Comment (kgaillot) [reply rpm/Makefile.am]: Hence the code in my first comment :)

**Review by kgaillot — APPROVED** [review]
Awesome :)

**Pull request merged**